be/test/exprs/hash_functions_test.cpp

```diff
@@ -139,6 +139,66 @@ TEST_F(HashFunctionsTest, test_xx_hash3_64) {
     }
 }
 
+#define INT128_LITERAL(high, low) (((int128_t)high << 64) | (uint64_t)low)
+
+TEST_F(HashFunctionsTest, test_xx_hash3_128) {
```

Comment:
Why not write some sql-test to cover this new function?

Reply:
No specific reason since there is already UT for xx_hash3_64.

I'll add some test cases later.